store/postgres/src/dynds/private.rs

```diff
@@ -24,7 +25,7 @@ pub(crate) struct DataSourcesTable {
     table: DynTable,
     block_range: DynColumn<sql_types::Range<Integer>>,
     manifest_idx: DynColumn<Integer>,
-    params: DynColumn<Jsonb>,
+    param: DynColumn<Bytea>,
```

Comment:
In all other places, we use `Binary` instead of `Bytea`

Reply:
Done